packages/as-sha256/test/perf/index.test.ts

```diff
@@ -1,12 +1,17 @@
-import {itBench} from "@dapplion/benchmark";
+import {itBench, setBenchOpts} from "@dapplion/benchmark";
 import * as sha256 from "../../src";
 
-// Aug 10 2021
+// Feb 2024 Mac M1
 // digestTwoHashObjects vs digest64 vs digest
-//     ✓ digestTwoHashObjects 50023 times                                    19.17366 ops/s    52.15488 ms/op        -       1151 runs   60.0 s
```

Comment:
Side-remark: previous benchmark were run for `60S`, maybe we want to align with that.